### ISSUE TITLE
Remove location and team selection from client upload 

### ISSUE THREAD

**Issue by isabelcshaw** [onaio/reveal-frontend#974]
This is for the MDA Point workflow around uploading clients. 

Right now, when the user clicks "Add new csv" on the client page, they are required to select a location and then a team before they are able to upload a csv of clients/students. 

We need to remove both of those drop downs. The user should only be given an interface to upload a csv. It should still run the same validation checks, but no drop downs are needed. 

The clients will be associated with locations via a field (school_ID) in the csv. And we will handle the team association elsewhere.

**Comment by craigappl** [comment]
@rkodev  I believe it would be a best practice to set the default teamId in the OpenSRP server instead of requiring it to be passed in. I know this is more work because you have to change the API, but think it's better in the long run and adds flexibility in the future in the case we want to add that to the CSV.

Therefore, this work is dependent on the changes by Ronald in the OpenSRP server.

**Comment by isabelcshaw** [comment]
@moshthepitt @craigappl based on my conversation with @rkodev earlier, this has been completed

**Comment by rkodev** [comment]
@craigappl I have updated the API, and it's currently on the reveal stage server. The teamId is no longer a requirement

**Comment by isabelcshaw** [comment]
@ciremusyoka adding the csv structure here: https://docs.google.com/spreadsheets/d/1uqX6BHxXb5M-3GHVTI2tbCKLMuNbF9hOC5iTHsSd41Y/edit#gid=0

Working on adding in some test date. A few notes on this: 
- I'm using the OpenSRP and Reveal IDs from a csv I downloaded from the page (from Emafusini Nazarene)
- I am not 100% certain on the format of the Eswatini national ID, but they asked us for that field to have 13 characters - can we keep the validation flexible/unrestricted? 
- I've added real school names for the pilot, found in [this spreadsheet shared with us ](https://docs.google.com/spreadsheets/d/1N5FaEDV8dj247ZPAIm9Ebio-0oyRQ2UZ/edit#gid=2092832762). My understanding is that we will be generating the unique IDs for each school in the system and then sharing those IDs with the team in Eswatini for them to use when they compile the rosters. @rkodev does that sound correct to you? Is there anything else needed here to generate those IDs?
